traffic_simulation.py

```diff
@@ -25,7 +25,7 @@ def random_intersection_placement(width: int, height: int) -> List[List[Intersec
 # Main logic for the Traffic Simulation 
 # Pygame should not be in this file
 class TrafficSimulation:
-    def __init__(self, num_of_cars, matrix: List[List[Intersection]] = random_intersection_placement(GRID_SIZE_X+1, GRID_SIZE_Y+1)):
+    def __init__(self, num_of_cars, matrix: List[List[Intersection]] = random_intersection_placement(GRID_SIZE_X, GRID_SIZE_Y)):
```

Comment:
Ok, I put the +1 here for a reason. Because the grid is actually one smaller when it's created

Reply:
yeah, i just put the +1 inside the random_intersection_placement method instead. still behaves the same way

Reply:
I see, looks good